modules/plugins/flexboxIE.js

```diff
@@ -24,4 +33,41 @@ export default function flexboxIE(
   if (alternativeProps.hasOwnProperty(property)) {
     style[alternativeProps[property]] = alternativeValues[value] || value
   }
+  if (property === 'flex') {
+    // For certain values we can do straight mappings based on the spec
+    // for the expansions.
+    if (flexShorthandMappings.hasOwnProperty(value)) {
+      style.msFlex = flexShorthandMappings[value]
+      return
+    }
+    // Here we have no direct mapping, so we favor looking for a
+    // unitless positive number as that will be the most common use-case.
+    if (isPositiveNumber.test(value)) {
+      style.msFlex = `${value} 1 0%`
+      return
+    }
+
+    // The next thing we can look for is if there are multiple values.
+    const flexValues = value.split(/\s/)
+    // If we only have a single value that wasn't a positive unitless
+    // or a pre-mapped value, then we can assume it is a unit value.
+    switch (flexValues.length) {
+      case 1:
+        style.msFlex = `1 1 ${value}`
+        return
+      case 2:
+        // If we have 2 units, then we expect that the first will
+        // always be a unitless number and represents flex-grow.
+        // The second unit will represent flex-shrink for a unitless
+        // value, or flex-basis otherwise.
+        if (isPositiveNumber.test(flexValues[1])) {
```

Comment:
Would `parseFloat(flexValues[1]) >= 0` not work here?

Reply:
I don't believe so, as for example `parseFloat('2px')` will equal `2`, and we are looking for unitless, since `parseFloat('2')` parses to the same thing and represents `flex-shrink`, while the former value of `2px` will represent `flex-basis`.

Reply:
I think this does call out an issue though in that it's not handling floats, so something like `flex: 1.1` should map to `flex: 1.1 1 0%`, which this implementation isn't doing. Will update the matching to account for floats, thanks for pointing this out!